src/config.js

```diff
@@ -125,13 +152,21 @@ const setConfig = async cfg => {
     config.wso2BearerToken = cfg.WS02_BEARER_TOKEN;
 };
 
-
 const getConfig = () => {
     return config;
 };
 
+// useful for closing the open handler of the fs.watch, especially in unit tests.
+const destroy = () => {
+    if (fsWatcher && typeof (fsWatcher.close) === 'function') {
```

Comment:
Why do we test type of `fsWatcher.close` here?

Reply:
Not a strict necessity, indeed. I just wanted to be sure that it is defined and it won't crash since the official documentation states for the method `close`:
```Stop watching for changes on the given fs.FSWatcher. Once stopped, the fs.FSWatcher object is no longer usable.```
And I had the impression that it won't even be defined after calling it, but now that I debug it I see that it is still there. Maybe I faced another issue during my tests, I am not sure. I can remove it.

Reply:
No harm leaving it, and it sounds like the docs aren't especially precise, so maybe we ought to leave it there. I just thought it was unusual.